scripts/examples/openiotsdk_example.sh

```diff
@@ -51,14 +51,12 @@ APP_VERSION_STR="0.0.1"
 declare -A tdb_storage_param=([instance]=sram [memspace]=0 [address]=0x0 [size]=0x100000)
 declare -A ps_storage_param=([instance]=qspi_sram [memspace]=0 [address]=0x660000 [size]=0x12000)
 
-readarray -t TEST_NAMES <"$CHIP_ROOT"/src/test_driver/openiotsdk/unit-tests/testnames.txt
-
-declare -a SUPPORTED_APP_NAMES
-SUPPORTED_APP_NAMES+=("shell")
-SUPPORTED_APP_NAMES+=("lock-app")
-SUPPORTED_APP_NAMES+=("all-clusters-app")
+readarray -t SUPPORTED_APP_NAMES <"$CHIP_ROOT"/examples/platform/openiotsdk/supported_examples.md
```

Comment:
I don't think this must be an MD file.

Reply:
I decided to use MD file because it is easily consumed in the documentation. The rest of the users (Vscode tasks and this script) can easily transform items to the required form.